componentens/Landingpage/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { use } from "react";
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function Landingpage() {
+  const [entry, setEntry] = useState("");
+  const [message, setMessage] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    setMessage(`${entry}`);
+    setEntry("");
+  };
+  return (
+    <>
+      <container>
+        {" "}
+        <Form onSubmit={handleSubmit}>
+          <p>Where is the journey going?</p>
```

Comment:
This could act as the label of the form (via aria-labelledby)

Reply:
this was not added as well